app/event/send-enrichment-error-event.js

```diff
@@ -1,7 +1,8 @@
 const raiseEvent = require('./raise-event')
+const { v4: uuidv4 } = require('uuid')
 
 const sendEnrichmentErrorEvent = async (paymentRequest, error) => {
```

Comment:
Can you create a unit test for this file?

https://github.com/DEFRA/ffc-pay-request-editor/blob/main/test/unit/event/send-enrich-request-blocked-event.test.js

Here's a copy-pasta guide

Reply:
Tests added.